collect_app/src/main/res/layout/chooser_list_layout.xml

```diff
@@ -51,21 +51,23 @@ the License.
 			android:layout_width="match_parent"
 			android:layout_height="match_parent"
 			android:text="@string/no_items_display_forms"
-			android:textSize="21sp"
-			android:paddingStart="20dip"
-			android:paddingLeft="20dp"
-			android:paddingEnd="20dip"
-			android:paddingRight="20dp"
+			android:textSize="@dimen/textsize_empty_list_alt_tv"
+			android:paddingStart="@dimen/padding_empty_list_alt_tv"
+			android:paddingLeft="@dimen/padding_empty_list_alt_tv"
+			android:paddingEnd="@dimen/padding_empty_list_alt_tv"
+			android:paddingRight="@dimen/padding_empty_list_alt_tv"
 			android:gravity="center"
-			android:textColor="@drawable/black" />
+			android:background="@color/background_empty_list_alt_tv"
+			android:textColor="@color/text_color_empty_list_alt_tv" />
+
 	</LinearLayout>
 
 	<!-- Side navigation drawer UI -->
 	<ListView
 		android:id="@+id/sortingMenu"
-		android:layout_width="250dp"
+		android:layout_width="@dimen/width_side_navigation_drawer"
 		android:layout_height="match_parent"
-		android:layout_gravity="end"
-		android:background="#DDDDDD"/>
+		android:layout_gravity="end|right"
```

Comment:
This doesn't seem related to this pull request and it also makes the lint check for right-to-left languages fail! `end` will be `right` in the context of a left-to-right language and `left` in the context of a right-to-left language.